pkg/virtual/workspaces/util/util.go

```diff
@@ -25,12 +25,12 @@ import (
 	"k8s.io/apiserver/pkg/registry/generic"
 	apistorage "k8s.io/apiserver/pkg/storage"
 
-	workspaceapi "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1alpha1"
+	workspaceapiv1beta1 "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1beta1"
 )
 
 // getAttrs returns labels and fields of a given object for filtering purposes.
 func getAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
-	workspaceObj, ok := obj.(*workspaceapi.ClusterWorkspace)
+	workspaceObj, ok := obj.(*workspaceapiv1beta1.Workspace)
```

Comment:
do we even need this cast? There are accessors for this purpose.

Reply:
Depends whether we want to be able to add additional selectable fields in the future, that might be workspace-specific fields (in the `workspaceToSelectableFields` method).